build/yamls/windows/ovs-containerd/conf/PostStart-AntreaAgent-Containerd.ps1

```diff
@@ -0,0 +1,12 @@
+$networkName = "antrea-hnsnetwork"
+$networks = Get-HnsNetwork
+$networkExists = $false
+foreach ($network in $networks) {
+    if ($network.Name -eq $networkName) {
+        $networkExists = $true
+        break
+    }
+}
+if ($networkExists) {
+    Enable-VMSwitchExtension -Name "Open vSwitch Extension" -VMSwitchName "antrea-hnsnetwork"
```

Comment:
Could you explain why this poststart and prestop are required when OVS runs as container?

Reply:
@tnqn  When the OVS container is deleted, the OVS userspace program can no longer provide userspace slow path traffic. If the OVS extension is not disabled on windows, host network will be unavailable in a short time. Therefore, it is necessary to disable the OVS extension and switch back to the host network in this case.

During the startup of a pod, there are two scenarios to consider. In the first scenario, when the hnsnetwork does not exist, the antrea agent can create the hnsnetwork and enable the OVS extension to take control of the OVS network. In this case, the post-start action does nothing. In the second scenario, if the hnsnetwork already exists on the host, the agent container needs to enable the ovs extension during startup to keep network traffic available because the antrea agent won't enable ovs extension anymore when hnswork exists. 

In the past, when OVS was running on the host, the userspace ovs service was always in running state, and this issue did not need to be considered.